packages/@okta/i18n/src/properties/login.properties

```diff
@@ -649,5 +649,8 @@ consent.required.privacyPolicy = Privacy Policy
 consent.required.consentButton = Allow Access
 consent.required.cancelButton = Don't Allow
 
+# Polling state
+polling.title = There are too many users trying to sign in right now. We will automatically retry in {0} seconds.
```

Comment:
We would need to handle 1 second and 2 seconds as well.

Reply:
the refresh will always be something that is between 2-5 seconds.